### DISCUSSION TITLE
Battery stuck at 50%- not charging

### DISCUSSION THREAD

**Discussion by troyboy27** [pgale/lineage_15.1_Installation_TSV#18] [General]
I just used the guide (thank you!) to install the custom ROM on my LTS; however, it's not charging. The battery icon isn't indicating charged, and it won't go beyond 50%. Any ideas?

**Comment by troyboy27** [comment]
Nevermind- duh- it doesn't have a battery! (Face plant)

**Reply by pgale** [reply]
Yeah you got it 🤣

You can also turn off the battery icon in the settings - just so it doesn't mock you 🤪

**Comment by 3rob3** [comment]
I think this is somewhat related, but I can't get Android screensavers to launch unless the USB cable is connected. I believe it's because it thinks it is charging on this port. Any ideas or workarounds for this?
